### PR TITLE
AnalyserVariable: hold onto the owning component reference.

### LINKED ISSUE #1183: Analyser: Not holding onto references
When analysing a model the analyser does not hold onto object references that it later needs when generating code. This results in generated code that cannot be simulated.

### PR DESCRIPTION
**Pull request by agarny**
Fixes #1183.

### CODE REVIEW

File: src/analyservariable.cpp [hunk 1]
```diff
@@ -16,9 +16,12 @@ limitations under the License.
 
 #include "libcellml/analyservariable.h"
 
+#include "libcellml/variable.h"
+
```
Comment (hsorby) [inline src/analyservariable.cpp]: This include should be below the system include.

**Review by hsorby — CHANGES_REQUESTED** [review]

File: src/analyservariable.cpp (hunk 1 above)
Comment (agarny) [reply src/analyservariable.cpp]: f24e786c

**Review by nickerso — APPROVED** [review]
looks good to me, assuming CI passes...

**Comment by agarny** [comment]
> looks good to me, assuming CI passes...

Phew, it still does! :)

**Review by hsorby — APPROVED** [review]

**Pull request merged**